tests/dialects/test_postgres.py

```diff
@@ -34,6 +34,7 @@ def test_postgres(self):
         self.validate_identity("SELECT EXTRACT(QUARTER FROM CAST('2025-04-26' AS DATE))")
         self.validate_identity("SELECT DATE_TRUNC('QUARTER', CAST('2025-04-26' AS DATE))")
         self.validate_identity("STRING_TO_ARRAY('xx~^~yy~^~zz', '~^~', 'yy')")
+        self.validate_identity("POSITION('sub' IN 'string')")
```

Comment:
I think Giorgos already touched on this, but let's make sure we're not adding redundant tests in future PRs. It's easy for them to creep in and checking whether a test exists already is usually as simple as a keyword search in the repo.